src/gui/properties/peerlistdelegate.h

```diff
@@ -73,6 +73,10 @@ class PeerListDelegate: public QItemDelegate {
         const bool hideValues = Preferences::instance()->getHideZeroValues();
         QStyleOptionViewItem opt = QItemDelegate::setOptions(index, option);
         switch(index.column()) {
+        case DOWNLOADING_PIECE:
```

Comment:
Wouldn't it be better to place `switch` handlers in same order as the `enum` declares them?